sunkit_pyvista/tests/test_plotting.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import pfsspy
+import pytest
+import pyvista
+from pfsspy import tracing
+from pfsspy.sample_data import get_gong_map
+
+import astropy.constants as const
+import astropy.units as u
+import sunpy.data.test as test
+import sunpy.map as smap
+from astropy.coordinates import SkyCoord
+from sunpy.coordinates import frames
+
+from sunkit_pyvista import SunpyPlotter
+
+pyvista.OFF_SCREEN = True
+
+
+@pytest.fixture
+def aia171_test_map():
+    return smap.Map(test.get_test_filepath('aia_171_level1.fits'))
+
+
+@pytest.fixture
+def plotter():
+    return SunpyPlotter()
+
+
+def test_plot_map_with_functionlity(aia171_test_map, plotter, verify_cache_image):
+    plotter.plot_map(aia171_test_map, clip_interval=(0, 99)*u.percent)
+    plotter.plot_solar_axis()
+
+    bottom_left = SkyCoord(30*u.deg, -10*u.deg,
+                           frame=frames.HeliographicStonyhurst,
+                           obstime=aia171_test_map.date)
+    plotter.plot_quadrangle(bottom_left=bottom_left, width=20*u.deg,
+                            height=60*u.deg, color='blue')
+
+    line = SkyCoord(lon=[180, 190, 200] * u.deg,
+                    lat=[0, 10, 20] * u.deg,
+                    distance=[1, 2, 3] * const.R_sun,
+                    frame='heliocentricinertial')
+    plotter.plot_coordinates(line)
+
+    coordinate = SkyCoord(30*u.deg, -10*u.deg,
+                          frame=frames.HeliographicStonyhurst,
+                          obstime=aia171_test_map.date)
+    plotter.plot_coordinates(coordinate, color='blue')
+
+    plotter.show(cpos=(0, 1, 0), before_close_callback=verify_cache_image)
+
+
+def test_field_lines_figure(aia171_test_map, plotter, verify_cache_image):
```

Comment:
Do we not want the map to show up with the field lines?

Reply:
Sorry I misread David's comment, I'll add that in